manifests/qpid.pp

```diff
@@ -18,12 +18,14 @@
     require   => Service['qpidd'],
     logoutput => true,
   } ~>
-  exec { 'bind katello entitlements queue to qpid exchange messages that deal with entitlements':
-    command   => "qpid-config --ssl-certificate ${client_cert} --ssl-key ${client_key} -b 'amqps://localhost:5671' bind event ${candlepin_event_queue} '*.*'",
-    onlyif    => "qpid-config --ssl-certificate ${client_cert} --ssl-key ${client_key} -b 'amqps://localhost:5671' queues ${candlepin_event_queue}",
-    path      => '/usr/bin',
-    require   => Service['qpidd'],
-    logoutput => true,
+  qpid::bind_event {'entitlement.created':
```

Comment:
@ekohl would it look like this example from the puppet docs with candlepin_event_queue being in the spot port is:

apache::vhost {'homepages':
  port    => 8081,
  docroot => '/var/www-testhost',
}